src/read.lisp

```diff
@@ -402,7 +402,8 @@ and formats used. The following arguments also control the behavior of the read.
                         ((:object-format *read-default-object-format*) *read-default-object-format*)
                         ((:float-format *read-default-float-format*) *read-default-float-format*)
                         ((:length *read-length*) *read-length*)
-                        ((:level *read-level*) *read-level*))
+                        ((:level *read-level*) *read-level*)
+                        ((:ht-test *read-default-ht-test*) *read-default-ht-test*))
```

Comment:
Let's call the key `:hash-table-test`. Also, please update the documentation string.

Reply:
Done. Also updated docstring for read-json.